packages/webdriverio/src/context.ts

```diff
@@ -50,6 +58,13 @@ export class ContextManager {
             ) {
                 this.#currentContext = undefined
             }
+
+            /**
+             * Keep track of the context to which we switch
+             */
+            if (this.#browser.isMobile && event.command === 'switchContext') {
```

Comment:
We don't have to do it in this PR but we should incorperate the Appium Bidi event that notifies us on context changes https://github.com/appium/appium/issues/20741